about.html

```diff
@@ -0,0 +1,92 @@
+            </ul>
+        </nav>
+    </header>
+    <main>
+
+        <section class="main-top">
+            <div class="background">
+               <div class="message">
+                  <h1>international Fitness  <br> Submit conference <br> 2022</h1>
+                 <p class="fit">The must attend business festival for health, fitness, wellness, hospitality, spa and HR executives is back to become LIVE and IN-PERSON. Experience an immersive multi-day thought leadership conference, exhibition, start-up showcase and industry awards dinner, with multiple networking events, product launches, activation events, site visits and workouts across the week to showcase new trends, products, concepts, technologies, innovation, start-ups, franchises and emerging investment opportunities.</p>
+                  <p class="date">Please contact us per email for any further questions about international fitness  <br></p>
+                 <h4>thefitsummit.com <i class="fa-solid fa-angle-right"></i></h4>
+               </div>
+            </div>
+          </section>
+          <section class="sponsor-main">
+                <h2>international Fitness submit logo</h2>
+                <p>the logo of international fitness was decided through the logo competition 2021 from 10th august to 10th septmber.
+                    after online public voting and selection commitee judging process. the logo won the price and it is used for our submit through out the years
+                </p>
+
+                <img src="asset/fit-removebg-preview.png" alt="">
+          </section><hr>
+          <section class="logo">
+            <div class="top-list">
+            <h3>see the past fitness submit</h3>
+            <p>Take a look at the last two international fitness submit which took place at india </p>
+        </div>
+            <div id="start">
+                
+                    <div class="cont">
+                    <div class="one">    
+                         <h3>2017</h3>
+                        <p>internation submit in india</p>
+                     </div>
+                    </div>
+                <div class=" two">
+                    <h3>2018</h3>
+                    <p>internation submit in india</p>
+                </div>
+            </div>
+          </section>
+    
+    </main>
+    <footer>
+        <div class="partners">
+            <h2 class="partnerhead section-title">Partner</h2>
+            <div class="partner-logo">
+                <img src="asset/butter-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Conspicuous-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/fitbit-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Muscle-Lab-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/technogym-1-removebg-preview.png" alt="">
+            
+            </div>
+        </div>
+    </footer>
```

Comment:
### MOBİLE VERSİON
- [ ] Footer has two sections in the original kindly add it.
- [ ] Also partner section icons moving to the next line. Kindly change it.

![image](https://user-images.githubusercontent.com/38635158/175400013-b436d109-44aa-4d30-82b3-e37e95016454.png)


### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/175400080-dbe59143-2e93-44cb-98cf-95e38f03dc87.png)